apps/minifront/src/state/dutch-auction/index.ts

```diff
@@ -93,6 +105,7 @@ export const createDutchAuctionSlice = (): SliceCreator<DutchAuctionSlice> => (s
       await planBuildBroadcast('dutchAuctionSchedule', req);
 
       get().dutchAuction.setAmount('');
+      void get().dutchAuction.loadAuctionInfos();
```

Comment:
I realized that we should be reloading auctions after successfully scheduling an auction, so that the latest data shows up in the auctions list.

Reply:
Could this potentially give them a success, but still show the same auctions screen? Aka, should we await this instead?